metadata-ingestion/docs/sources/powerbi/powerbi_pre.md

```diff
@@ -21,7 +21,7 @@
 | `Page`                | `Chart`             |
 
 - If `Tile` is created from report then `Chart.externalUrl` is set to Report.webUrl.
-- The `Page` information is unavailable for PowerBI PaginatedReport.
+- The `Page` is unavailable for PowerBI PaginatedReport.
```

Comment:
this is still confusing - let's add PaginatedReport to the concept mapping table which will clean things up

also, how is Report.webUrl mapped to a chart if the report is mapped to a dashboard?

Reply:
The tile contains visualizations that can be created using a dataset, a text box, or by pinning a report. If a report is pinned within the tile, the `externalUrl` will point to that report.

Reply:
I see